1426430702844	Sync.Service	INFO	Loading Weave 1.38.0
1426430702846	Sync.Engine.Clients	DEBUG	Engine initialized
1426430702847	Sync.Engine.Clients	DEBUG	Resetting clients last sync time
1426430702852	Sync.Engine.Bookmarks	DEBUG	Engine initialized
1426430702856	Sync.Engine.Forms	DEBUG	Engine initialized
1426430702859	Sync.Engine.History	DEBUG	Engine initialized
1426430702863	Sync.Engine.Passwords	DEBUG	Engine initialized
1426430702865	Sync.Engine.Prefs	DEBUG	Engine initialized
1426430702868	Sync.Engine.Tabs	DEBUG	Engine initialized
1426430702869	Sync.Engine.Tabs	DEBUG	Resetting tabs last sync time
1426430702874	Sync.Engine.Addons	DEBUG	Engine initialized
1426430702875	Sync.Service	INFO	Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:36.0) Gecko/20100101 Firefox/36.0
1426430702878	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1426430702879	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430702880	Sync.Service	DEBUG	Caching URLs under storage user base: https://sync-186-us-west-2.sync.services.mozilla.com/1.5/21274104/
1426430702880	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430702880	Sync.AddonsReconciler	INFO	Registering as Add-on Manager listener.
1426430702880	Sync.AddonsReconciler	DEBUG	Adding change listener.
1426430702881	Sync.Tracker.History	INFO	Adding Places observer.
1426430702903	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430702904	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430702905	FirefoxAccounts	DEBUG	got keyPair
1426430702905	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1426430706905	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430706906	Sync.Service	DEBUG	User-Agent: Firefox/36.0.1 FxSync/1.38.0.20150306140238.
1426430706906	Sync.Service	INFO	Starting sync at 2015-03-15 22:45:06
1426430706907	Sync.Service	DEBUG	In sync: should login.
1426430706907	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430706907	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1426430785944	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
1426430787149	Sync.Tracker.History	DEBUG	Saving changed IDs to history
1426430807609	Sync.Tracker.History	DEBUG	Saving changed IDs to history
1426430811644	Sync.Tracker.History	DEBUG	Saving changed IDs to history
1426430824615	Sync.Tracker.History	DEBUG	Saving changed IDs to history
1426430883464	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 637"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:637 < _onComplete()@resource://services-common/hawkclient.js:194 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883466	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 653"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:653 < _onComplete()@resource://services-common/hawkclient.js:197 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883468	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 687"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:687 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:285 < _onComplete()@resource://services-common/hawkclient.js:204 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883469	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 687"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:687 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:285 < _onComplete()@resource://services-common/hawkclient.js:205 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883471	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 653"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:653 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:108 < _onComplete()@resource://services-common/hawkclient.js:210 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883472	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 637"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:637 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:109 < _onComplete()@resource://services-common/hawkclient.js:210 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883474	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 637"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:637 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:110 < _onComplete()@resource://services-common/hawkclient.js:210 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883475	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 687"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:687 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:113 < _onComplete()@resource://services-common/hawkclient.js:210 < onComplete()@resource://services-common/hawkclient.js:251 < onStopRequest()@resource://services-common/rest.js:461 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:145 < onNotify()@resource://gre/modules/services-sync/service.js:981 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1003 < sync/<()@resource://gre/modules/services-sync/service.js:1249 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1245 < <file:unknown>
1426430883475	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"message":null,"code":null,"errno":null}
1426430883475	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"message":null,"code":null,"errno":null}
1426430883476	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: null: null
1426430883476	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1426430883476	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1426430883476	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1426430883477	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
